composer/trainer/trainer.py

```diff
@@ -529,6 +530,17 @@ class Trainer:
 
         console_stream (TextIO | str, optional): The stream to write to. If a string, it can either be
             ``'stdout'`` or ``'stderr'``. (default: :attr:`sys.stderr`)
+        console_log_interval (int | str | Time, optional): Specifies how frequently to log metrics to console.
```

Comment:
imo lets only make this an arg to console logger and make default progress bars only

Reply:
so the user has to construct a `ConsoleLogger` object?